app/models/chef.rb

```diff
@@ -1,4 +1,11 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :dish_ingredients, through: :dishes
+   has_many :ingredients, through: :dish_ingredients
+
+   def unique_ingredients
+      ingredients.distinct.pluck(:name)
```

Comment:
Nice work! Adding the additional association makes this logic much easier. Great job remembering the `distinct`!